docs/gateway/manta.md

```diff
@@ -0,0 +1,56 @@
+# Minio Manta Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+Minio Gateway adds Amazon S3 compatibility to Manta Object Storage.
+
+## Run Minio Gateway for Manta Object Storage
+### Using Docker
+```
+docker run -p 9000:9000 --name azure-s3 \
```

Comment:
There's a lot of mentions of Azure throughout what appear to be Manta-specific files.